sdk/lib/atl/atlimage.h

```diff
@@ -555,26 +550,29 @@ class CImage
         // convert the file name string into Unicode
```

Comment:
Is the "TODO & FIXME" in this function still needed? (in the other functions I've seen you removed it)

Reply:
No, unneeded. Deleted.